src/quantum/azext_quantum/__init__.py

```diff
@@ -8,6 +8,12 @@
 import azext_quantum._help  # pylint: disable=unused-import
 
 
+# This is the version reported by the CLI to the service when submitting requests.
+# This should be in sync with the extension version in 'setup.py', unless we need to
+# submit using a different version.
+CLI_REPORTED_VERSION = "0.5.0"
```

Comment:
Can we use one variable? `VERSION` is also defined in `setup.py`.

Reply:
I agree we should consolidate, for now, this is a way to have the version in development in `master` reporting a certain value without triggering the release to happen.